FlaxEngine/Rendering/IDrawable.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2012-2019 Wojciech Figat. All rights reserved.
+
+namespace FlaxEngine.Rendering
+{
+    public interface IDrawable
```

Comment:
Could you add some documentation comments?

Reply:
Added